zulipterminal/model.py

```diff
@@ -1164,15 +1164,16 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
                       self.controller, -1)
 
     def formatted_local_time(self, timestamp: int,
-                             *, show_seconds: bool = False,
+                             *, show_seconds: bool,
                              show_year: bool = False) -> str:
         local_time = datetime.datetime.fromtimestamp(timestamp)
-        if show_seconds:
-            return local_time.strftime('%a %b %d %H:%M:%S')
-        if show_year:
-            return local_time.strftime('%a %b %-d %Y %-H:%M:%S')
-        else:
-            return local_time.strftime('%a %b %d %H:%M')
+        format_codes = (
```

Comment:
It's a good practice to make intermediate variable names context dependent, in general. Something like "format_date" could work?